### ISSUE TITLE
Journal view is empty

### ISSUE THREAD

**Issue by LukasRos** [monicahq/monica#2072]
As of this morning, the journal view is empty for me. I'm using the hosted version on https://app.monicahq.com/ and I have tracked a few activities (no manual journal entries). These entries still appear inside the contacts they're associated with but the journal view shows none.
![bildschirmfoto 2018-11-20 um 10 49 55](https://user-images.githubusercontent.com/192050/48765281-1c6c6600-ecb2-11e8-9e8f-856e019ffa5f.png)

**Comment by asbiin** [comment]
Is the ouptut of https://app.monicahq.com/journal/entries empty as well?

**Comment by LukasRos** [comment]
That URL shows me the following error:
`Trying to get property 'name' of non-object`

**Comment by djaiss** [comment]
Ah. Sorry about that @LukasRos 

I'll investigate.

**Comment by LukasRos** [comment]
I did a bit of testing of my own with a local installation and my SQL export.

The error message comes from line 27 in `app/Http/Resources/Contact/ContactShort.php`.

What I found is that I have a contact whose _gender_id_ is set to **0**. This probably was the result of API testing where I found the bug #1951.

Since I was able to find out which contact it was I could adjust their gender from within the Monica frontend and now the journal view works again.
